File: 2733/2733.cpp
class Solution {
public:
    int neitherMinimumNorMaximum(TreeNode* root) {
        if(!root) return 0;
        return 1+max(neitherMinimumNorMaximum(root->left),neitherMinimumNorMaximum(root->right));
    }
};

File: 2733/2733.cs
public class Solution {
    public int NeitherMinimumNorMaximum(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(NeitherMinimumNorMaximum(root.left), NeitherMinimumNorMaximum(root.right));
    }
}